jekyll/_cci2/pipeline-variables.md

```diff
@@ -111,29 +114,18 @@ curl -u ${CIRCLECI_TOKEN}: -X POST --header "Content-Type: application/json" -d
 ### Passing parameters when triggering pipelines using the CircleCI web app
 {: #passing-parameters-when-triggering-pipelines-using-the-circleci-web-app }
 
-In addition to using the CLI and API, you can also trigger a pipeline with parameters from the CircleCI web app. To do this:
-
-  1. Navigate to the dashboard view in the web app.
-  2. Use the project filter to select the desired project.
-  3. Use the branch filter to select the branch on which you want to run the new pipeline.
-  4. Click the **Trigger Pipeline** button (towards the top right corner of the page).
-  5. Use the **Add Parameters** dropdown to specify the type, name, and value of your desired parameters.
-  6. Click **Trigger Pipeline**.
+In addition to using the API, you can also trigger a pipeline with parameters from the CircleCI web app. If you pass a parameter when triggering a pipeline from the web app, and the parameter has not been declared in the configuration file, the pipeline will fail with the error `Unexpected argument(s)`.
 
-**NOTE:** If you pass a parameter when triggering a pipeline from the web app, and the parameter has not been declared in the configuration file, the pipeline will fail with the error `Unexpected argument(s)`)
-
-
-## The scope of pipeline parameters
-{: #the-scope-of-pipeline-parameters }
+1. Use the project filter to select the desired project
```

Comment:
Nit: Since these are complete sentences could we end each list item with a period?